CDP4Common/Helpers/NestedElementTreeGenerator.cs

```diff
@@ -94,6 +94,45 @@ public IEnumerable<NestedElement> Generate(Option option, DomainOfExpertise doma
             return createNestedElements;
         }
 
+        /// <summary>
+        /// Returns the <see cref="NestedParameter"/>s in a flat list
+        /// </summary>
+        /// <param name="option">
+        /// The <see cref="Option"/> for which the <see cref="NestedElement"/> tree is created. When the <see cref="Option"/>

```

Comment:
The docs state a NestedElement tree is created, looking at the code this is not the case. Please update docs, or code